libraries/ms-common/src/stm32f0xx/chip_id.c

```diff
@@ -0,0 +1,23 @@
+#include <stdint.h>
+
+#include "chip_id.h"
```

Comment:
Nitpick: the import order should have the header file you're implementing first, so these includes should be swapped